tests/qos/test_tunnel_qos_remap.py

```diff
@@ -583,8 +583,19 @@ def test_pfc_watermark_extra_lossless_standby(ptfhost, fanouthosts, rand_selecte
         # Ingress packet from uplink port
         testutils.send(ptfadapter, src_port, pkt, 1)
         # Get the actual egress port
-        result = testutils.verify_packet_any_port(ptfadapter, exp_pkt, dst_ports)
```

Comment:
normally, we should extend the timeout in any_port function here.

Reply:
have you tried with extend the timout value for verify_packet_any_port?

Reply:
I believe I did. Ultimately though if the packet has already arrived then any non-zero timeout especially a default should be valid. The situation with verify_packet_any_port vs verify_packet in the description just by itself indicates the former has some issues. 
Technically this PR only addresses one spot where this lockup occurs. If there's a bug in PTF it also needs to be diagnosed and addressed.